context/app/api/client.py

```diff
@@ -139,6 +140,14 @@ def get_vitessce_conf(self, entity):
                 ]
             }
         else:
-            vc = get_view_config_class_for_data_types(entity=entity, nexus_token=self.nexus_token)
-            conf = vc.build_vitessce_conf()
-            return conf
+            try:
+                vc = get_view_config_class_for_data_types(
+                    entity=entity, nexus_token=self.nexus_token
+                )
+                conf = vc.build_vitessce_conf()
+                return conf
+            except Exception:
+                current_app.logger.info(
+                    f'Building vitessce conf threw error: {traceback.format_exc()}'
+                )
+                return {}
```

Comment:
- You will need to make sure you're getting the error reports from IEC if we do this... If they can't promise to run a grep every day for two weeks, I don't think we should do this.
- I don't think this should be left in indefinitely: File a followup issue, and let's take it out in the next sprint.